<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/logo/WhatsApp Image 2022-10-28 at 19.50.34.jpeg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>WhatsApp Image 2022-10-28 at 19.50.34.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>501</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnRegistrarPedidos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>5816221.png</normaloff>5816221.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnRegistrarMesa">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>4813181.png</normaloff>4813181.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Registrar
 pedidos </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Registrar
 pedidos de Mesa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPersonal">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>2361718.png</normaloff>2361718.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Registro
 del personal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVentas">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>312101.png</normaloff>312101.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Registro
 de ventas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Registro
 de horario</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnHorario">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>182832.png</normaloff>182832.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tblPedidosMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>751</width>
      <height>192</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>Registro</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DNI</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cod_venta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Direccion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Producto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Bebidas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>tabla-madera-sobremesa-vacia-fondo-borroso_1253-1584.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCliente">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cliente</string>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>label</zorder>
   <zorder>frame</zorder>
   <zorder>tblPedidosMenu</zorder>
   <zorder>btnCliente</zorder>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
